prepare_heatmap_matrix.py

```
@@ -0,0 +1,82 @@
+            matrix[algo_idx][bin_idx] = dup_rate
+
+    # Write matrix format for gnuplot
+    with open('cdc_heatmap_matrix.txt', 'w') as f:
+        # Write column headers (chunk sizes in KB)
+        f.write("# ")
+        for bin_key in sorted_bins:
+            f.write(f"{bin_key/1024.0:.1f} ")
+        f.write("\n")
+
+        # Write data rows (one per algorithm)
+        for algo_idx, algo_name in enumerate(algorithms):
```

Ruff (0.14.14)

[warning] Loop control variable `algo_name` not used within loop body

(B007)